README.md

```diff
@@ -1,13 +1,13 @@
 Dash Core staging tree 0.14.1
-===============================
+=============================
```

Comment:
Why don't we just add a `#` at the beginning of this header and remove the `===` entirely? It achieves the same effect in Markdown and doesn't require re-aligning the `===` underneath every time.

Reply:
sounds like another pr to me :)

Reply:
`#` and `====` are used interchangeably in various other `*.md` files all over the repo. Should swap them all at once if we decide to do so imo, but I would rather just leave it as is tbh.

Reply:
Agree this is a nit, and swapping them all at once makes sense.